charts/wharf-helm/values.yaml

```
@@ -101,6 +101,33 @@ web:
   # listens on. [Read more (kubernetes.io/docs)](https://kubernetes.io/docs/concepts/services-networking/service/#defining-a-service)
   containerPort: 8080
 
+  # -- Settings for the OpenId Connect authentication login.
+  oidc:
```

The README.md contains the full `web.oidc` object formatted as JSON in its `default` field. You can override this by specifying a custom string to be used instead when rendering the markdown:

```suggestion
  # -- Settings for the OpenId Connect authentication login.
  # @default -- see the `web.oidc.*` fields for more info
  oidc:
```